planetary-social/nos
# #1594: Create Account screen and new feature flag

## Issues covered
#1594 

## Description
The Create Account screen, plus a feature flag.

## How to test
1. Navigate to Settings
2. Enable the New Onboarding Flow feature flag
3. Log out
4. Work through the new flow
5. Observe the Create Account view when you get there

## Screenshots/Video

https://github.com/user-attachments/assets/2a1db838-eeb6-451b-be98-84f7f4e7fe65
